packages/SwingSet/src/vats/comms/state.js

```diff
@@ -50,7 +50,7 @@ function makeSyscallStore(syscall) {
 }
 
 function commaSplit(s) {
-  if (s === undefined || s === '') {
+  if (!s || s === '') {
```

Comment:
Looks like `!s` is to handle the null case, but it also handles the empty string case.
```suggestion
  if (!s) {
```

Reply:
Oops, thanks, I'm perpetually confused about JS's truthyness rules. In this case I think I'll add a comment for others like me. The function might get `undefined` (because the DB key that feeds it was not present, and `undefined` is the obvious return value from a `.get(missingKey)`, or `null` (because the DB lookup *actually* happened on the other side of a JSON-encoded vat-worker-to-kernel-process pipe and the JSON decoder produces `null` instead of `undefined`), or `''` (because the key is present but was populated with `slots.join(',')` and `slots` was empty). In all three cases, and empty list is the correct return value.